### PR TITLE
Coadd the psf, including dithers, using the stack

### LINKED ISSUE #17: figure out how to make a psf coadd with recenter using the stack
Should be possible to do a tiny coadd.

If not we can pull in ours from pizza

### PR DESCRIPTION
**Pull request by esheldon**
closes #17

### CODE REVIEW

**Comment by esheldon** [comment]
Needed to retrigger travis so it can go out and get the changes in master of descwl-shear-sims

**Comment by beckermr** [comment]
travis is really slow right now

**Comment by esheldon** [comment]
I'm still running more jobs but this appears to fix the bias we saw with dithers, which was m ~ 0.01
```
m1: 0.000130674 +/- 0.00249878 (3 sigma)
```

File: descwl_coadd/coadd.py [hunk 1]
```diff
@@ -134,11 +142,13 @@ def _make_exps(self):
...
                 weight = se_obs.weight.array
 
-                psf_image = se_obs.get_psf(
+                psf_gsimage, psf_offset = se_obs.get_psf(
                     pos.x,
                     pos.y,
-                    center_psf=True,
-                ).array
+                    center_psf=False,
+                    get_offset=True,
```
Comment (beckermr) [inline descwl_coadd/coadd.py]: So this now matches what we are doing in pizza cutter, right?

Comment (esheldon) [reply descwl_coadd/coadd.py]: Yes

**Comment by beckermr** [comment]
ohhh woops. you already merged! 🤦‍♂

**Pull request merged**